packages/create-cloudflare/e2e-tests/frameworks.test.ts

```diff
@@ -475,7 +475,6 @@ function getFrameworkTests(opts: {
 					},
 				],
 				testCommitMessage: true,
-				quarantine: true,
```

Comment:
we should start adding comments when we quarantine tests

Reply:
we should also evaluate why tests still need to be quarantined (maybe an investigation + adding respective comments) would be a good start 🙂

Reply:
PS: as far as I can tell quarantined tests get forgotten and their templates can completely break without anyone noticing, I wonder if we could increase the visibility of these too

Reply:
(that's actually what happened here isn't it? 🤔 next broke but no one noticed right away since it's quarantined)

Reply:
that is correct yes. and I agree that we should evaluate what tests are quarantined and why. I took a quick look and both `astro` and `analog` tests are quarantined. I have context over the `analog` ones but not the `astro` ones